main.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const app = require("electron");
+const app = require("electron").app;
```

Comment:
I'd rather use

```javascript
const { app } = require("electron")
```

Reply:
I'm not so familiar with JS, but I did what I did just to be consistent with the imports below.
Should I update those imports as well?

Reply:
yeah you should be able to do something like

```javascript
const { start, lastWindow } = require("./index")
```

Reply:
Done
Could you please check